projects/aca-content/src/lib/components/dl-custom-components/thumbnail-column/thumbnail-column.component.html

```diff
@@ -1,2 +1,10 @@
 <mat-icon class="adf-datatable-selected" *ngIf="isSelected" svgIcon="selected"></mat-icon>
-<img *ngIf="!isSelected" class="adf-datatable-center-img-ie" [src]="thumbnailUrl" [alt]="tooltip" [matTooltip]="tooltip" />
+<ng-container *ngIf="!isSelected">
+  <ng-container *ngIf="isIcon; else image">
+    <mat-icon class="adf-datatable-center-icon">{{thumbnailUrl}}</mat-icon>
+  </ng-container>
+  <ng-template #image>
+    <img *ngIf="!isSelected" class="adf-datatable-center-img-ie" [src]="thumbnailUrl" [alt]="tooltip" [title]="tooltip" />
```

Comment:
Do we need additional `!isSelected` check here? It's check on the parent ng-container

Reply:
cleaned up template, also removed the custom css classes that seem to be doing anything for this layout